plugins/serverless/src/tasks/deploy.ts

```diff
@@ -0,0 +1,49 @@
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { hookFork, styles } from '@dotcom-tool-kit/logger'
+import { Task } from '@dotcom-tool-kit/types'
+import { ServerlessSchema } from '@dotcom-tool-kit/types/lib/schema/serverless'
+import { VaultEnvVars } from '@dotcom-tool-kit/vault'
+import { spawn } from 'child_process'
+
+export default class ServerlessDeploy extends Task<typeof ServerlessSchema> {
+  static description = 'Deploys on AWS'
+
+  async run(): Promise<void> {
+    const { useVault, configPath } = this.options
```

Comment:
**nitpick**: which variables are deconstructed from `this.options` are a little inconsistent here, we can probably extract the `buildNumVariable` and `region` variables too. it was an oversight in the `provision.ts` file too as I added the `buildNumVariable` option later on 😅